.gitignore

```diff
@@ -8,4 +8,7 @@
 
 # development 
 /logs
-/venv
\ No newline at end of file
+/venv
+
+# env
+.env
```

Comment:
Question: What kind of env variables you keep inside?

Reply:
Good point. Will update the docs.